pkgs/by-name/sk/skeditor/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildDotnetModule,
+  fetchFromGitHub,
+  dotnetCorePackages,
+}:
+buildDotnetModule rec {
+  pname = "skeditor";
+  version = "2.8.1";
+
+  src = fetchFromGitHub {
+    owner = "skeditorteam";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-ojE276nonX52UTjvdKL5mX8dj3MNElqlR1A/c0vT9WE=";
+  };
+
+  projectFile = "SkEditor/SkEditor.csproj";
+  executables = [ "SkEditor" ];
+  nugetDeps = ./nuget-deps.nix;
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+
+  passthru.updateScript = ./update.sh;
+
+  meta = with lib; {
+    description = "SkEditor is an app for editing Skript files";
```

Comment:
```suggestion
    description = "App for editing Skript files";
```

Reply:
I would reword app to be "Skript Code Editor for Skript" but I just pulled that description but I guess that works for now.

Reply:
Or even better "An IDE for Skript".